### ISSUE TITLE
Deleting namespaces

### ISSUE THREAD

**Issue by elafont** [zero-os/0-stor#468]
* I have a User1 who is the owner of test260.0stor organization (all tests done with zstor)
* I create a new namespace eraseme
```
$ zstor namespace create eraseme  
INFO[0000] namespace "eraseme" created        
```
* User1 has all the rights on eraseme
```
$ zstor namespace permission get User1  eraseme
Read: true
Write: true
Delete: true
Admin: true
```
* When trying to delete the namespace eraseme, or any other created 
```
$ zstor namespace delete  eraseme  
INFO[0000] namespace "eraseme" deleted 
```

* All seems right, but going to the IYO Web site shows that *eraseme* is still live and working 
```
$ zstor file upload aFile  -n eraseme 
INFO[0000] Setting namespace config to [eraseme]        
INFO[0000] data from "aFile" uploaded as key = "aFile"
```

No matter how many times I ask for a delete, the namespace is never erased.

The config files used are the sames as in #467.

**Comment by lafont-e** [comment]
Small code to test, only uses itsyouonline client
```go
// test code for  itsyouonline issue 690: deleting a namespace

package main

import (
	"fmt"
	iyo "github.com/zero-os/0-stor/client/itsyouonline"
	"log"
	"os"
)

const (
	user1        = "lafonte@greenitglobe.com"
	user1Id      = "otzMc8WXFwBqdfU_YiPiMQBgTkeU"
	user1Secret  = "S_ZiZNGWZOe_7ZzkTdbkmCV_ZiZp"
	organization = "test260"
)

var (
	iyo1 = iyo.Config{
		Organization:      organization,
		ApplicationID:     user1Id,
		ApplicationSecret: user1Secret}
)

func usage() {
	log.Fatalf("Usage: %s [create|delete] <namespace> ", os.Args[0])
}

func createNs(iyoCl *iyo.Client, ns string) {
	fmt.Println("Creating namespace ", ns)
	err := iyoCl.CreateNamespace(ns)
	if err != nil {
		log.Fatalf("Error creation of namespace %q failed: (%s)", ns, err)
	}

	fmt.Printf("namespace %q created\n", ns)
	fmt.Println()
}

func removeNs(iyoCl *iyo.Client, ns string) {
	fmt.Println("Removing namespace ", ns)
	err := iyoCl.DeleteNamespace(ns)
	if err != nil {
		log.Fatalf("Error removing of namespace %q failed: (%s)", ns, err)
	}

	fmt.Printf("namespace %q removed\n", ns)
	fmt.Println()
}

func main() {
	if len(os.Args) < 3 {
		usage()
	}

	iyoCl, err := iyo.NewClient(iyo1)
	if err != nil {
		log.Fatal("Error creating IYO client: ", err)
	}

	ns := os.Args[2] // 2nd parameter is namespace to operate

	switch os.Args[1] {
	case "create":
		createNs(iyoCl, ns)
	case "delete":
		fmt.Print("1st Try.... ")
		removeNs(iyoCl, ns)
		fmt.Print("2nd Try (Should fail) .... ")
		removeNs(iyoCl, ns)
	default:
		usage()
	}
}

```

**Comment by lafont-e** [comment]
moved to (https://github.com/itsyouonline/identityserver/issues/690) and solved there

**Comment by GlenDC** [comment]
This issue is only to be closed once the vendored IYO dependency has been updated, and it can be confirmed that with this update the issue, as described here, has been fixed.

**Comment by lafont-e** [comment]
the vendor dependency has been updated , but the code that matters seems the same even after the update
```
make update_dep DEP="github.com/itsyouonline/identityserver"
```

$ vim github.com/zero-os/0-stor/vendor/github.com/itsyouonline/identityserver/clients/go/itsyouonline/organizations_service.go
```
// Deletes an organization and all data linked to it (join-organization-invitations, oauth_access_tokens, oauth_clients, logo)
    814 func (s *OrganizationsService) DeleteOrganization(globalid string, headers, queryParams map[string]interface{}) (*http.Response, error) {
    815         var err error
    816 
    817         resp, err := s.client.doReqNoBody("GET", s.client.BaseURI+"/organizations/"+globalid, headers, queryParams)
```

The GET method should be a DELETE

Not sure if after the dependency update I should do somehting more....

**Comment by GlenDC** [comment]
@lafont-e you really shouldn't share secret information such an IYO `app_id`/`app_secret`. Removed it from your comment, but just so you know for next time...

**Comment by GlenDC** [comment]
Closed in 30014ae.
